test/parallel/test-path.js

```diff
@@ -561,8 +561,9 @@ if (common.isWindows) {
                '\\\\?\\' + process.cwd().toLowerCase() + '\\foo\\bar');
   assert.equal(path.win32._makeLong('foo/bar').toLowerCase(),
                '\\\\?\\' + process.cwd().toLowerCase() + '\\foo\\bar');
-  assert.equal(path.win32._makeLong('C:').toLowerCase(),
-               '\\\\?\\' + process.cwd().toLowerCase());
+  const currentDeviceLetter = path.parse(process.cwd()).root.substring(0, 2);
+  assert.equal(path.win32._makeLong(currentDeviceLetter).toLowerCase(),
+                 '\\\\?\\' + process.cwd().toLowerCase());
```

Comment:
This last line was likely an oversight: I see only 2 added spaces, it was correct before.